### PR TITLE
Fix some PVS-Studio warnings

### PR DESCRIPTION
**Pull request by dimhotepus**
Product page: https://www.viva64.com/en/pvs-studio/
How to use for free: https://www.viva64.com/en/b/0600/

### CODE REVIEW

**Comment by dimhotepus** [comment]
@Xottab-DUTY May be you find this one also interesting enough to merge?

**Comment by Xottab-DUTY** [comment]
@dimhotepus, hi!
Sorry for not merging for a long time. The main reason is that I was afraid of changes that could change the behaviour and thus break the compatibility with the master-servers which still use original sources.

Could you send this pull request to UniSpy, so that more experienced in GameSpy SDK people can properly review it?
https://github.com/GameProgressive/UniSpySDK

File: src/GameSpy/GP/gpiKeys.c [hunk 1]
```diff
@@ -172,7 +172,7 @@ GPResult gpiSaveKeysToBuffer(GPConnection *connection, char **buffer)
 		gsDebugFormat(GSIDebugCat_GP, GSIDebugType_Memory, GSIDebugLevel_HotError, "gpiSaveKeysToBuffer: buffer Out of memory.");
 		Error(connection, GP_MEMORY_ERROR, "Out of memory.");
 	}
-	bytesWritten = sprintf(*buffer, keysHeader);
+	bytesWritten = sprintf(*buffer, "%s", keysHeader);
```
Comment (Xottab-DUTY) [inline src/GameSpy/GP/gpiKeys.c:175]: Be aware that this change was also provided in https://github.com/OpenXRay/GameSpy/pull/6 and merged. :)

**Comment by Xottab-DUTY** [comment]
Thank you for sending these changes to UniSpy.

I think, it's better to just use UniSpy instead of supporting this repository.
So, I'm closing this pull request and I hope we will migrate to UniSpy someday. 🙃 
https://github.com/OpenXRay/xray-16/issues/260

**Pull request closed without merging**